### PR TITLE
DM-11117: Retrieve HiTS 2015 data

### PR DESCRIPTION
**Pull request by djreiss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,30 @@
 # verify_ap_hits2015
-Data from HiTS (2015) to verify the performance of alert production in the LSST DM stack.
 
-This repo is designed to be used as a template for developing new data sets for integration into `verify_ap`.
+Data from HiTS (2015) to verify the performance of alert production in the LSST DM stack.
 
-It is currently set up for using `obs_test`.
+Contains DeCam data from the HiTS (2015) fields `Blind15A_26`, `Blind15A_40`, and `Blind15A_42`.
```
Comment (kfindeisen) [inline README.md]: Nit-picky, but it's DECam. Same spelling error elsewhere in the file.

File: ups/verify_ap_hits2015.table [hunk 2]
```diff
@@ -0,0 +1 @@
+setupRequired(obs_decam)
```
Comment (kfindeisen) [inline ups/verify_ap_hits2015.table:1]: This goes beyond my knowledge of EUPS, but don't you also need a `.cfg` file? Most packages have one.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,30 @@
...
 
 Relevant Files and Directories
 -----
 path                  | description
 :---------------------|:-----------------------------
-`raw`                 | To be populated with photometrically and astrometrically calibrated data. Currently contains a single small fits file (taken from `obs_test`) to test `git-lfs` functionality.
-`calib`               | To be populated with master calibs. Currently empty.
+`raw`                 | Photometrically and astrometrically calibrated compressed DeCam fits images from the HiTS (2015) fields `Blind15A_26`, `Blind15A_40`, and `Blind15A_42`.
+`calib`               | DeCam master calibs and `wtmap`s from the 2015 HiTS campaign. No raw images. See below for filename information.
```
Comment (kfindeisen) [inline README.md]: Do you still need the `README.md` in `calib/`?

File: README.md [hunk 4]
```diff
@@ -1,21 +1,30 @@
...
 `templates`           | To be populated with calibrated images intended to be used as templates. Currently empty.
```
Comment (kfindeisen) [inline README.md]: Are templates a separate ticket?

**Review by kfindeisen — APPROVED** [review]
Looks good, aside from some omissions that might be deliberate. Please clean up your commit history (especially the "Fix..." commits) before merging.

File: README.md (hunk 4 above)
Comment (djreiss) [reply README.md]: Templates will be coadds, which Eric is working on right now.

File: ups/verify_ap_hits2015.table (hunk 2 above)
Comment (djreiss) [reply ups/verify_ap_hits2015.table:1]: Likewise my knowledge of EUPS but this package is modeled off of [validation_data_cfht](https://github.com/lsst/validation_data_cfht), which has no `.cfg` file.

File: README.md (hunk 3 above)
Comment (djreiss) [reply README.md]: Probably not, although I suppose some of the details from the main README could go here.

**Comment by RobertLuptonTheGood** [comment]
.cfg files are used by sconsUtils, they are nothing to do with eups.

**Pull request merged**